src/Install/Installer.cpp

```diff
@@ -39,30 +39,33 @@ namespace sua {
 
         _installerAgent->installBundle(input);
 
-        bool     installing                  = true;
-        uint32_t count                       = 0;
+        uint32_t tries                       = 0;
         int32_t  progressPercentage          = 0;
         int32_t  progressNotificationLimiter = 0;
 
-        while(installing) {
+        while(_installerAgent->installing()) {
             progressPercentage = _installerAgent->getInstallProgress();
-						std::this_thread::sleep_for(2000ms);
-            count++;
-            if(progressPercentage >= 100 || count >= 120) {
-                installing = false;
+            std::this_thread::sleep_for(2000ms);
+            tries++;
+            if(tries >= 120) {
```

Comment:
optional: same, constant could be introduced

Reply:
Ok